jenkins/clean_project.py

```diff
@@ -69,10 +69,20 @@ def main(project, days, filt, delete):
       description='Delete old instances from a project')
   parser.add_argument('--project', help='Project to clean', required=True)
   parser.add_argument(
-      '--days', type=float, help='Clean items more than --days old', required=True)
+      '--days', type=int,
+      help='Clean items more than --days old (added to --hours)')
+  parser.add_argument(
+      '--hours', type=float,
+      help='Clean items more than --hours old (added to --days)')
   parser.add_argument(
       '--delete', action='store_true', help='Really delete things when set')
   parser.add_argument(
-      '--filter', default='name:tmp*', help='Filter down to these instances')
+      '--filter', default="name~'tmp.*' AND NOT tags.items:do-not-delete",
+      help='Filter down to these instances')
   args = parser.parse_args()
-  main(args.project, args.days, args.filter, args.delete)
+
+  if args.days is None and args.hours is None:
```

Comment:
Please either note that you're explicitly trying to distinguish from `--days=0` otherwise use an implicit false here (`if not args.days`)

https://google.github.io/styleguide/pyguide.html?showone=True/False_evaluations#True/False_evaluations